lib/RestClient.ts

```diff
@@ -227,6 +229,7 @@ export class RestClient {
                         response.result = obj;
                     }
                 }
+                response.headers = res.message.headers;
```

Comment:
if (res.message.headers) ?

Reply:
I think this is [guaranteed to have the headers property](https://nodejs.org/api/http.html#http_class_http_incomingmessage) so I think we can just expect it (like we do with status code above).